src/resources/auth/controller.js

```diff
@@ -1,7 +1,6 @@
-const prisma = require('../..//utils/dbClient')
+const prisma = require('../../utils/dbClient')
 const bycript = require('bcrypt')
-const createToken = require('../../utils/authentification');
-const crypto = require('crypto');
+const {createToken} = require('../../utils/authentification');
```

Comment:
No need to change this, but the correct spelling here is `authentication`

Reply:
thx
will change to appropriate spelling as normal
<img src="https://media4.giphy.com/media/pKEufUXBqsLi8/giphy.gif"/>